plugins/modules/fusion_az.py

```diff
@@ -172,15 +188,10 @@ def main():
 
     fusion = get_fusion(module)
     state = module.params["state"]
-    azone = get_az(module, fusion)
-    if not get_region(module, fusion):
-        module.fail_json(
-            msg="Region {0} does not exist.".format(module.params["region"])
-        )
 
-    if not azone and state == "present":
```

Comment:
If the resource already exists, removing this check and running fusion_az with state="present" will cause it to fail and print an error. That is not wanted in this case, it should only print green "ok" and do nothing.